src/main/groovy/io/gatling/gradle/GatlingPluginExtension.groovy

```diff
@@ -2,6 +2,51 @@ package io.gatling.gradle
 
 class GatlingPluginExtension implements JvmConfigurable {
 
+    private static final String API_TOKEN_PROPERTY = "gatling.enterprise.apiToken"
+    private static final String API_TOKEN_ENV = "GATLING_ENTERPRISE_API_TOKEN"
+
+    static class Enterprise {
+        private String apiToken
+        private UUID packageId
+        private URL url = new URL("https://cloud.gatling.io/api/public")
+
+        def url(String url) {
```

Comment:
better stick to the MOJO notation: `setUrl`

Reply:
https://github.com/gatling/gatling-gradle-plugin/pull/37#discussion_r699465190